### ISSUE TITLE
Allow failures of builds using MOAB `master` and `develop` branches.

### ISSUE THREAD

**Issue by pshriwise** [svalinn/DAGMC#847]
As I recall, these builds are present to notify us of any changes upstream in MOAB that might break DAGMC builds/tests.

We currently require these to pass in CI, which I think is a little overkill. IMO it's sufficient to require that our builds/tests pass and Docker images publish successfully only for official releases of MOAB. I'm curious if @shimwell may have a need for Docker images using MOAB's `develop` and `master` branches?

It's still very useful to have these builds of course, so I'm going to look into allowing failures for these builds but always running them for merges into our `develop` branch.

**Comment by shimwell** [comment]
I don't have any specific need for Docker images with develop or master versions of MOAB.

**Comment by ahnaf-tahmid-chowdhury** [comment]
I think we are ready to close the issue since it has been solved by PR #899

**Comment by gonuke** [comment]
I agree that this is no longer relevant and can be closed.  I'll let @pshriwise confirm....
